index.js

```diff
@@ -56,7 +64,9 @@ function average (benchmarkResults) {
   const results = {}
   ;['firstInstall', 'repeatInstall', 'withWarmCacheAndLockfile', 'withWarmCache', 'withLockfile']
```

Comment:
you use this same array inside `generateSvg`, so I guess it can be moved to a separate file